src/bin/flamegraph.rs

```diff
@@ -9,9 +9,9 @@ use flamegraph::Workload;
 #[derive(Debug, Parser)]
 #[clap(version)]
 struct Opt {
-    /// Profile a running process by pid
-    #[clap(short, long)]
-    pid: Option<u32>,
+    /// Profile a running process by pid (comma separated list)
+    #[clap(short, long, value_delimiter(','))]
+    pid: Option<Vec<u32>>,
```

Comment:
Can we avoid the `Option` wrapper by using the default for that case?

Reply:
Behaviour of `Option<Vec<T>>` and `Vec<T>` is the same for clap, so I just removed the `Option`